app/controllers/assessments_controller.rb

```diff
@@ -54,8 +54,15 @@ def self.documentation_for_get
   end
 
   def show
+        # throwing a fake error to test that sentry works
+    raise MyCustomException.new "Message, message, message", "Yup"
+      rescue MyCustomException => e
```

Comment:
Should this still be here?


I think we can have a formal test for this in the test suite  - give me a shout if you need a hand.